__tests__/users.ts

```diff
@@ -587,7 +587,7 @@ describe('query readHistory', () => {
   it('should not authorize when not logged in', () =>
     testQueryErrorCode(client, { query: QUERY }, 'UNAUTHENTICATED'));
 
-  it("should return user's reading history in descending order", async () => {
+  it("should return user's reading history", async () => {
```

Comment:
Make sure to run lint fix, you use double quotes instead of single

Reply:
I think the reason this passed is due to the apostrophe being used in the actual text. Let me rephrase it to avoid the usage of double-quotes.